src/html.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const fontsBasePath = '/fonts';
+
+const fontsPaths = [];
```

Comment:
Let's add comments here to all changes that are different from the default gatsby version.